src/sentry/tasks/groupowner.py

```diff
@@ -13,11 +13,8 @@
 from sentry.utils.committers import get_event_file_committers
 
 PREFERRED_GROUP_OWNERS = 2
-OWNER_CACHE_LIFE = 3600  # seconds
-PREFERRED_GROUP_OWNER_AGE = timedelta(days=1)
-GROUP_PROCESSING_DELAY = timedelta(
-    minutes=60
-)  # Minimum time between processing the same group id again
+OWNER_CACHE_LIFE = 604800  # 1 week in seconds
```

Comment:
Just checked, we had 17mm groups that were active in the last week, so that's roughly the number of items that will live in the cache. That seems like it won't be too bad, although it might actually make sense here to abbreviate the cache key name just in the interests of memory.

Maybe just something like
```
# Abbreviation for "workflow-owners-ingestion:group-{}"
cache_key = "w-o-i:g-{}".format(event.group_id)
```

No one really interacts with these keys, so they don't need to be too readable.